src/index.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Calculator from './components/Calculator';
+import './App.css';
+import './index.css';
```

Comment:
- [ ] As per the project requirements, you are required to delete all the boilerplate text, images, and styles that the CRA created.

![image](https://user-images.githubusercontent.com/86318284/189194828-4803a7d7-47dc-4306-

Reply:
I resolved the issue, thank you @Nemwel-Boniface